components/Header/index.js

```diff
@@ -1,31 +1,34 @@
 import styled from "styled-components";
+import Link from "next/link";
 
 export default function Header() {
   return (
-    <StyledHeader>
-      <StyledH1>Movie-Box</StyledH1>
+    <StyledHeader href="/">
+      <Title>Movie-Box</Title>
     </StyledHeader>
   );
 }
 
-const StyledHeader = styled.header`
+const StyledHeader = styled(Link)`
```

Comment:
See above
```suggestion
const StyledHeader = styled.header`
```

If you adjust your code accordingly, some of the current styles might become obsolete, please check :)